### ISSUE TITLE
Luminosity ratio for the "Usage Data" toggle button is not meeting the standard non-text contrast ratio

### ISSUE THREAD

**Issue by Vineela479** [material-components/material-components-android#2036]
Description:
Foreground to background ratio for the Talkback 'OFF' state toggle button from the default Android settings is 1.542:1

Expected behavior: 
The non-text contrast ratio for the Talkback 'OFF' state toggle button from the default Android settings should be greater than or equal to 3.0:1

Android API version: 30

Device: Pixel 3a
![Toggle button-off state-Android settings](https://user-images.githubusercontent.com/78538706/107043463-11e20c80-67e9-11eb-98fd-55b4a56f1a1e.png)

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by drchen** [comment]
Switches have shadows so there's actually no contrast ratio issues.
